pkg/cmd/repo/fork/fork.go

```diff
@@ -225,25 +226,13 @@ func forkRun(opts *ForkOptions) error {
 			}
 		}
 		if remoteDesired {
-			remoteName := "origin"
-
+			remoteName := opts.RemoteName
 			remotes, err := opts.Remotes()
 			if err != nil {
 				return err
 			}
 			if _, err := remotes.FindByName(remoteName); err == nil {
-				renameTarget := "upstream"
-				renameCmd, err := git.GitCommand("remote", "rename", remoteName, renameTarget)
```

Comment:
This clearly broke existing functionality 😭 No longer capable of renaming remote to `upstream`